ajax/libs/skeletonic/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "skeletonic",
+  "version": "1.0.89",
+  "description": "Skeletonic is a lightweight, intuitive and ultra-responsive CSS framework to speed up your Digital and Mobile Web development needs.",
+  "filename": "skeletonic.min-1.0.89.css",
+  "author": "Reedia Limited",
+  "homepage": "https://skeletonic.io",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/reedia/skeletonic"
+  },
+  "keywords": [
+    "skeletonic",
+    "skeletonic.min-1.0.89.css",
```

Comment:
I think we should remove this because it's too specific. `keywords` here will help user find this library not the version :slightly_smiling_face:

Reply:
Agreed. This has been removed on v1.0.93 at https://github.com/reedia/skeletonic/releases/tag/v1.0.93